main.js

```diff
@@ -1,9 +1,27 @@
 const buttons = document.querySelectorAll("button");
-const screen = document.getElementById("calculation_screen");
+let user_input = "";
+let calc_done = false;
 
 buttons.forEach(function (btn) {
   btn.addEventListener("click", (e) => {
     const button_clicked = e.currentTarget.textContent;
-    console.log(button_clicked);
+    if (calc_done && button_clicked !== "C") {
+      user_input = button_clicked;
+      document.querySelector("input").value = user_input;
+      calc_done = false;
+    } else if (button_clicked == "=") {
+      if (user_input !== "") {
```

Comment:
What happens if I click 3 = = 3 on the calculator?  How would you get around the user accidentally pressing the = sign too many times? or any of the other signs, i.e. 6 - - - - - - 6?